scripts-available/CDB_Groups.sql

```diff
@@ -0,0 +1,124 @@
+-- Creates a new group
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Group_CreateGroup(group_name text)
```

Comment:
As far as I remember only the organization owner should be responsible for creating groups, right? So function execution should be granted to that user/role.